### PR TITLE
[bitnami: redis] Adds sizeLimit for redis-tmp-conf volume of replica component in redis chart

### PR DESCRIPTION
**Pull request by vijay2308**
Signed-off-by: Vijay Kumar [vanamvijay23@gmail.com](mailto:vanamvijay23@gmail.com)

**Description of the change**
 Adds `sizeLimit` for` redis-tmp-conf `volume of replica component in redis chart

**Benefits**
Will be able to limit the size of `redis-tmp-conf` which is present in `master` already 

**Possible drawbacks**
No 

**Applicable issues**
fixes #

~**Additional information**~

### CODE REVIEW

**Comment by carrodher** [comment]
Thanks for creating this PR. Please, note it is needed to sign the commits. Your signature certifies that you wrote the patch or otherwise have the right to contribute the material. You can find more info about this topic, in [the “Sign your work” section from the contributing guidelines](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work).

**Comment by carrodher** [comment]
It seems you added more commits than expected. Could you rebase from master and only add the changes related to Redis?

**Comment by vijay2308** [comment]
> It seems you added more commits than expected. Could you rebase from master and only add the changes related to Redis?

its done!

File: bitnami/redis/Chart.yaml [hunk 1]
```diff
@@ -24,4 +24,4 @@ maintainers:
 name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
-version: 16.8.9
+version: 16.8.10
```
Comment (FraPazGal) [inline bitnami/redis/Chart.yaml:27]: Thanks for the PR @vijay2308! Could you please modify the version and bump its minor. Even though it is small, we are adding a new feature.

```suggestion
version: 16.9.0
```

Comment (vijay2308) [reply bitnami/redis/Chart.yaml]: done!

**Comment by vijay2308** [comment]
@fredleger  can i get this reviewed pls ?

**Comment by fredleger** [comment]
i guess wrong tag in user @vijay2308 ? I'm not a maintainer her :-)

**Comment by vijay2308** [comment]
@FraPazGal can i get review on this please ?

**Comment by fmulero** [comment]
Sorry @vijay2308 the changes look good, but some tests are failing. We are investigating

**Comment by vijay2308** [comment]
> Sorry @vijay2308 the changes look good, but some tests are failing. We are investigating

is there any ETA on the same ?

**Review by fmulero — APPROVED** [review]
LGTM

**Pull request merged**